### PR TITLE
Update FPWD from feedback

### LINKED ISSUE #154: Exact URL-s for the JsonSchema2023 and CredentialSchema2023
At the moment, these terms are defined as

- `https://www.w3.org/ns/credentials/issuer-dependent#JsonSchema2023`
- `https://www.w3.org/ns/credentials/issuer-dependent#CredentialSchema2023`

In my view, that is wrong. The `issuer-dependent` namespace (resulting from the `@vocab` introduced in the VCDM `@context` as a "fall-back") was not meant to express core properties.

This specification is on standard track, and therefore the terms should be placed as terms (to be more precise, RDF Classes, because they are instances of `type` statements) alongside the VCDM terms. This means the correct URL-s should be

- `https://www.w3.org/2018/credentials#JsonSchema2023`
- `https://www.w3.org/2018/credentials#CredentialSchema2023`

And both terms should be properly added to the [VCDM Vocabulary](https://w3c.github.io/vc-data-model/vocab/credentials/v2/vocabulary.html)

---

To be clear: this issue should not necessary delay the FPWD publication of the spec. This type of change can be done on new versions later.

cc @msporny @dlongley

### PR DESCRIPTION
**Pull request by decentralgabe**
Addresses feedback in https://github.com/w3c/transitions/issues/507

Also fix #154

***
<a href="https://pr-preview.s3.amazonaws.com/w3c/vc-json-schema/pull/155.html" title="Last updated on May 18, 2023, 5:33 PM UTC (3fcae53)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/vc-json-schema/155/7eaa9b9...3fcae53.html" title="Last updated on May 18, 2023, 5:33 PM UTC (3fcae53)">Diff</a>

### CODE REVIEW

**Review by andresuribe87 — APPROVED** [review]

**Review by OR13 — APPROVED** [review]

File: FPWD/2023-05-23/index.html [hunk 1]
```diff
@@ -252,6 +252,33 @@
         "name": "W3C"
       }
     },
+    {
+      "id": "https://w3c-ccg.github.io/data-integrity-spec/",
```
Comment (OR13) [inline FPWD/2023-05-23/index.html:256]: this is outdated, but can be fixed later... see the references to it in https://github.com/w3c/vc-di-bbs/blob/main/index.html#L100

File: FPWD/2023-05-23/index.html [hunk 2]
```diff
@@ -252,6 +252,33 @@
...
+      "type": "TechArticle",
+      "name": "Data Integrity. Manu Sporny; Dave Longley. Credentials Community Group. CG-DRAFT",
+      "url": "https://w3c-ccg.github.io/data-integrity-spec/",
+      "publisher": {
+        "name": "W3C"
+      }
+    },
+    {
+      "id": "https://www.w3.org/TR/vc-jwt/",
```
Comment (OR13) [inline FPWD/2023-05-23/index.html:265]: same comment here... https://github.com/w3c/vc-di-bbs/blob/main/index.html#L100

File: FPWD/2023-05-23/index.html [hunk 3]
```diff
@@ -252,6 +252,33 @@
...
+      "type": "TechArticle",
+      "name": "Securing Verifiable Credentials using JSON Web Tokens",
+      "url": "https://www.w3.org/TR/vc-jwt/",
+      "publisher": {
+        "name": "W3C"
+      }
+    },
+    {
+      "id": "https://json-schema.org/draft/2020-12/json-schema-validation.html",
```
Comment (OR13) [inline FPWD/2023-05-23/index.html:274]: see https://github.com/transmute-industries/vc-json-schema/blob/main/index.html#L28

File: FPWD/2023-05-23/index.html (hunk 1 above)
Comment (decentralgabe) [reply FPWD/2023-05-23/index.html:256]: updating now

File: FPWD/2023-05-23/index.html (hunk 2 above)
Comment (decentralgabe) [reply FPWD/2023-05-23/index.html:265]: don't see this one?

**Comment by TallTed** [comment]
Merged too fast! Commenting inline..

File: FPWD/2023-05-23/index.html [hunk 4]
```diff
@@ -138,10 +138,10 @@
 .removeOnSave{display:none}
 }
 </style>
-<meta name="description" content="Among other things, the [VC-DATA-MODEL] specifies the models used for Verifiable Credentials,
+<meta name="description" content="Among other things, the [VC-DATA-MODEL-2] specifies the models used for Verifiable Credentials,
           Verifiable Presentations, and explains the relationships between three parties:
           issuers, holders, and verifiers. Critical pieces of functionality referenced
-          throughout the [VC-DATA-MODEL] are the that of of verifiability, extensibility, and semantic
+          throughout the [VC-DATA-MODEL-2] are the that of verifiability, extensibility, and semantic
```
Comment (TallTed) [inline FPWD/2023-05-23/index.html:144]:
```suggestion
          throughout the [VC-DATA-MODEL-2] are verifiability, extensibility, and semantic
```

File: FPWD/2023-05-23/index.html [hunk 5]
```diff
@@ -138,10 +138,10 @@
...
           interoperability. This specification provides a mechanism to make use of a Credential Schema in
```
Comment (TallTed) [inline FPWD/2023-05-23/index.html:145]: or
```suggestion
          issuers, holders, and verifiers. Verifiability, extensibility, and semantic
          interoperability are critical pieces of functionality referenced throughout
          the [VC-DATA-MODEL-2]. This specification provides a mechanism to make use
          of a Credential Schema in
```

Comment (decentralgabe) [reply FPWD/2023-05-23/index.html:145]: thanks @TallTed added here https://github.com/w3c/vc-json-schema/pull/156

**Pull request merged**